libafl/src/mutators/numeric.rs

```diff
@@ -0,0 +1,473 @@
+//! Mutators for integer-style inputs
```

Comment:
int mutators or numeric mutators? We should stick to one name I think

Reply:
Or would float mutators end up here as well?

Reply:
> Or would float mutators end up here as well?

That was my idea. I don't know what would make sense there and I don't need them right now, but they may in the future.

On the other hand, we can always rename the file. What do you prefer, I don't mind.

Reply:
Same, no strong preference

Reply:
Let's leave it for now then.